app/src/lib/stores/pull-request-store.ts

```diff
@@ -388,16 +388,7 @@ export class PullRequestStore extends TypedBaseStore<GitHubRepository> {
       })
     }
 
-    if (prsToInsert.length <= 0) {
-      return
-    }
-
     return this.pullRequestDatabase.transaction('rw', table, async () => {
-      // since all PRs come from the same repository
-      // using the base repoId of the fist element
-      // is sufficient here
-      const repoDbId = prsToInsert[0].base.repoId!
```

Comment:
We already define `repoDbId` at the start of the function, so this is unnecessary work

Reply:
Good catch!